common/ethnicities/wc_ethnicities_orcish_rework.txt

```diff
@@ -0,0 +1,448 @@
+        1 = { name = eye_depth_neg    range = { 0.56 0.56  }    }
+	}
+	gene_eye_angle = {
+        1 = { name = eye_angle_neg    range = { 0.25 0.6  }    }
+	}
+	gene_eye_height = {
+        1 = { name = eye_height_neg    range = { 0.2 0.4  }    }
+	}
+	gene_eye_distance = {
+        1 = { name = eye_distance_neg    range = { 0.45 0.65 }    }
+	}
+	gene_eye_shut = {
+        1 = { name = eye_shut_neg    range = { 0.4 0.8  }    }
+	}
+	gene_bs_eye_corner_depth = {
+        1 = { name = eye_corner_depth_neg    range = { 0 1 }    }
+		1 = { name = eye_corner_depth_pos    range = { 0 1 }    }
+	}
+	gene_bs_eye_fold_shape = {
+        1 = { name = eye_fold_shape_pos    range = { 0 1 }    }
+		1 = { name = eye_fold_shape_neg    range = { 0 1 }    }
+	}
+	gene_bs_eye_size = {
+        1 = { name = eye_size_pos    range = { 0 1 }    }
+		1 = { name = eye_size_neg    range = { 0 1 }    }
+	}
+	gene_bs_eye_upper_lid_size = {
+        1 = { name = eye_upper_lid_size_pos    range = { 0 1 }    }
+	}
+	face_detail_eye_lower_lid_def = {
+        1 = { name = eye_lower_lid_def    range = { 0.5 1 }    }
+	}
+	face_detail_eye_socket = {
+        1 = { name = eye_socket_01    range = { 0 1 }    }
+		1 = { name = eye_socket_02    range = { 0 1 }    }
+		1 = { name = eye_socket_03    range = { 0 1 }    }
+	}
+	
+	###Tusks
+	gene_bs_mouth_tusks = {
+        1 = { name = mouth_fangs   range = { 1 1	  }    }
+    }
+	static_tusks = {
+        1 = { name = orcish_tusks    range = { 0 1	  }    }
+    }
+
+	#Body
+	gene_bs_body_type = {
+		10 = { name = body_fat_head_fat_low		range = { 0.6 0.7 } }
+	}
+	gene_bs_body_shape = {
+		10 = { name = body_shape_triangle_half		range = { 0.7 1 } }
+	}
+	
+	###Other	
+	hairstyles = {
+		1 = { name = orcish_hairstyles 		range = { 0.0 1.0 } }
+	}
+	beards = {
+		1 = { name = orcish_beards 		range = { 0.0 1.0 } }
+	}
+	gene_eyebrows_shape = {
+        25 = {  name = no_eyebrows             range = { 0 1 }     }
+		1 = {  name = avg_spacing_avg_thickness             range = { 0.4 1 }     }
+	}
+	gene_eyebrows_fullness = {
+        25 = {  name = no_eyebrows             range = { 0 1 }     }
+		1 = {  name = layer_2_avg_thickness            range = { 1 1 }     }
+	}
+	gene_body_hair = {
+        1 = {  name = body_hair_sparse             range = { 0 0.4 }     }
+	}
+	face_detail_temple_def = {
+        1 = {  name = temple_def             range = { 0.2 0.8 }     }
+	}
+	expression_brow_wrinkles = {
+        1 = {  name = brow_wrinkles_01             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_02             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_03             range = { 0.5 1 }     }
+		1 = {  name = brow_wrinkles_04            range = { 0.5 1 }     }
+	}
+	expression_eye_wrinkles = {
+        1 = {  name = eye_wrinkles_01             range = { 0 1 }     }
+		1 = {  name = eye_wrinkles_02             range = { 0 1 }     }
+		1 = {  name = eye_wrinkles_03             range = { 0 1 }     }
+	}
```

Comment:
Don't you want to give this gene? 
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/107958212-224d7080-6fbb-11eb-9c23-1a2909210223.png)

</details>
It will boost nasolabials.

Reply:
You mean `cheek_wrinkles_both`? I'll add it.

Reply:
> You mean `cheek_wrinkles_both`? I'll add it.

Yes.